playbooks/openshift-master/private/config.yml

```diff
@@ -13,29 +13,14 @@
 
 - import_playbook: certificates.yml
 
-- name: Disable excluders
-  hosts: oo_masters_to_config
-  gather_facts: no
-  roles:
-  - role: openshift_excluder
-    r_openshift_excluder_action: disable
-
-- name: Gather and set facts for master hosts
+- name: Disable excluders and gather facts
   hosts: oo_masters_to_config
   pre_tasks:
```

Comment:
Given that the only role we have in this play is openshift_facts, I'd say we could do away with pre_tasks, roles, and post_tasks and just have tasks:.

Reply:
Good idea.